### PR TITLE
Add raven authentication and lambda

### PR DESCRIPTION
**Pull request by PedroGCEtges**

### CODE REVIEW

**Comment by PedroGCEtges** [comment]
[[Documentation] Add Raven](https://trello.com/c/VfOD95pA/1450-documentation-add-raven)

File: Leaf API.postman_collection.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"info": {
-		"_postman_id": "a28e15a6-4088-49e1-9337-fb6578fe6d41",
+		"_postman_id": "7e8a32e3-3111-4586-9a40-e0475666261a",
```
Comment (fabyan-alexander) [inline Leaf API.postman_collection.json:3]: You don't need to include this change 😉

File: Leaf API.postman_environment.json [hunk 2]
```diff
@@ -186,9 +186,39 @@
...
+		},
+		{
+			"key": "raven_refresh_token",
+			"value": "(auto)",
+			"enabled": true
+		},
+		{
+			"key": "leaf_auth_raven",
+			"value": "raven-oauth2-helper.withleaf.io",
+			"enabled": true
```
Comment (fabyan-alexander) [inline Leaf API.postman_environment.json:218]: This property should follow the pattern for the other leaf auth helpers, which is to include the property as a postman collection property.

File: Leaf API.postman_environment.json [hunk 3]
```diff
@@ -186,9 +186,39 @@
 			"key": "trimble_application_name",
 			"value": "fill with your app's",
 			"enabled": true
+		},
+		{
+			"key": "raven_client_id",
+			"value": "fill with your app's",
```
Comment (fabyan-alexander) [inline Leaf API.postman_environment.json:192]: We can add a better description for this property
```suggestion
			"value": "fill with the Client ID obtained from Raven",
```

File: Leaf API.postman_environment.json [hunk 4]
```diff
@@ -186,9 +186,39 @@
...
+			"enabled": true
+		},
+		{
+			"key": "raven_client_secret",
+			"value": "fill with your app's",
```
Comment (fabyan-alexander) [inline Leaf API.postman_environment.json:197]: We can add a better description for this property
```suggestion
			"value": "fill with the Client Secret obtained from Raven",
```

File: Leaf API.postman_environment.json [hunk 5]
```diff
@@ -186,9 +186,39 @@
...
+		},
+		{
+			"key": "raven_code",
+			"value": "code after redirecting user",
+			"enabled": true
+		},
+		{
+			"key": "raven_refresh_token",
+			"value": "(auto)",
+			"enabled": true
```
Comment (fabyan-alexander) [inline Leaf API.postman_environment.json:213]: I don't see the command line in step 2 to set the raven_refresh_token.
This property is defined using JS to take the response received and set this property.
We can also add a better value for this.
```suggestion
			"key": "raven_refresh_token",
			"value": "Raven refresh token obtained from step 2 "Get Raven Token" (auto)",
			"enabled": true
```

File: Leaf API.postman_environment.json [hunk 6]
```diff
@@ -186,9 +186,39 @@
...
+		},
+		{
+			"key": "raven_client_secret",
+			"value": "fill with your app's",
+			"enabled": true
+		},
+		{
+			"key": "raven_redirect_url",
+			"value": "fill with your app's",
```
Comment (fabyan-alexander) [inline Leaf API.postman_environment.json:202]: We can add a better description for this property
```suggestion
			"value": "the allowed redirect URL defined with Raven",
```

File: Leaf API.postman_environment.json [hunk 7]
```diff
@@ -186,9 +186,39 @@
...
+			"enabled": true
+		},
+		{
+			"key": "raven_code",
+			"value": "code after redirecting user",
```
Comment (fabyan-alexander) [inline Leaf API.postman_environment.json:207]:
```suggestion
			"value": "code inside the redirect url after authenticating the user in Raven website(auto)",
```

File: Leaf API.postman_environment.json [hunk 8]
```diff
@@ -1,5 +1,5 @@
 {
-	"id": "c581b7cc-4573-4a97-bd4e-7f7b82c85dd4",
+	"id": "87cf34fd-92a3-4ac4-88df-c5d4bc68b57c",
```
Comment (fabyan-alexander) [inline Leaf API.postman_environment.json:2]: We don't need to commit this change, please revert it

**Review by fabyan-alexander — CHANGES_REQUESTED** [review]
Welcome to Postman 😄 
Sorry if there are a lot of comments to review, but these changes are quite easy to fix.
You also need to update your documentation from the main branch as there are some conflicts to fix.
Call me if you need help

File: Leaf API.postman_environment.json (hunk 2 above)
Comment (PedroGCEtges) [reply Leaf API.postman_environment.json]: Done.

File: Leaf API.postman_environment.json (hunk 3 above)
Comment (PedroGCEtges) [reply Leaf API.postman_environment.json]: Done.

File: Leaf API.postman_environment.json (hunk 4 above)
Comment (PedroGCEtges) [reply Leaf API.postman_environment.json:197]: Done.

File: Leaf API.postman_environment.json (hunk 5 above)
Comment (PedroGCEtges) [reply Leaf API.postman_environment.json]: Done.

File: Leaf API.postman_environment.json (hunk 7 above)
Comment (PedroGCEtges) [reply Leaf API.postman_environment.json]: Done.

**Review by fabyan-alexander — APPROVED** [review]
Great work!

**Pull request merged**